devlib/collector/ftrace.py

```diff
@@ -240,6 +241,9 @@ def available_functions(self):
         return self.target.read_value(self.available_functions_file).splitlines()
 
     def reset(self):
+        # Save kprobe events
+        kprobe_events = self.target.read_value(self.kprobe_events)
```

Comment:
Out of curiosity, do you have a sample of what reading the file provides with a probe set ? The kernel doc does not seem to describe that, it only talks about the file behavior when written to (I may have missed it).

Reply:
Sorry if I misunderstood, but is this what you asked for?
```Bash
$ echo 'p do_sys_open' > /sys/kernel/tracing/kprobe_events                                                                                                                                                                                                                       
$ echo 'r do_sys_open' >> /sys/kernel/tracing/kprobe_events                                                                                                                                                                                                                      
$ cat /sys/kernel/tracing/kprobe_events                                                                                                                                                                                                                                          
p:kprobes/p_do_sys_open_0 do_sys_open
r18:kprobes/r_do_sys_open_0 do_sys_open
```

Reply:
Makes sense, thanks